.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    allow:
+      - dependency-type: production
+    directory: /
+    schedule:
+      interval: weekly
+    versioning-strategy: widen
+
+  - package-ecosystem: npm
+    allow:
+      - dependency-type: development
+    directory: /
+    schedule:
+      interval: weekly
+    versioning-strategy: increase
```

Comment:
Updates `devDependencies` on a weekly schedule using the `increase` version strategy

> Leave the constraint if the original constraint allows the new version, otherwise, bump the constraint.